package.json

```diff
@@ -16,48 +16,47 @@
     "url": "git+https://github.com/tech-matters/serverless.git"
   },
   "devDependencies": {
-    "@types/jest": "^25.2.3",
-    "@types/jsonwebtoken": "^8.5.0",
-    "@types/lodash": "^4.14.172",
-    "@types/node-fetch": "^2.5.5",
-    "@types/swagger-ui-express": "^4.1.1",
-    "@types/twit": "^2.2.28",
+    "@types/jest": "^28.1.7",
+    "@types/jsonwebtoken": "^8.5.8",
+    "@types/lodash": "^4.14.184",
+    "@types/node-fetch": "^2.6.2",
+    "@types/swagger-ui-express": "^4.1.3",
+    "@types/twit": "^2.2.30",
     "@types/uuid": "^8.3.4",
-    "@typescript-eslint/eslint-plugin": "^2.23.0",
-    "@typescript-eslint/parser": "^2.23.0",
+    "@typescript-eslint/eslint-plugin": "^5.33.1",
+    "@typescript-eslint/parser": "^5.33.1",
     "eslint": "^6.8.0",
     "eslint-config-airbnb-base": "^14.1.0",
     "eslint-config-airbnb-typescript": "^7.0.0",
     "eslint-config-prettier": "^6.10.0",
     "eslint-import-resolver-typescript": "^2.0.0",
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-prettier": "^3.1.2",
-    "jest": "^26.0.1",
-    "jest-each": "^27.5.1",
+    "jest": "^28.1.3",
+    "jest-each": "^28.1.3",
     "jsonwebtoken": "^8.5.1",
     "mockdate": "^3.0.5",
-    "prettier": "^1.19.1",
-    "redoc-cli": "^0.10.2",
-    "ts-jest": "^26.0.0",
-    "twilio-run": "^3.2.2",
-    "typescript": "^3.8.3"
+    "prettier": "^2.7.1",
+    "redoc-cli": "^0.13.19",
+    "ts-jest": "^28.0.8",
+    "twilio-run": "^3.4.3",
+    "typescript": "^4.7.4"
   },
   "engines": {
     "node": "^12.0"
   },
   "dependencies": {
-    "@tech-matters/serverless-helpers": "^2.0.0",
-    "@twilio-labs/serverless-runtime-types": "^1.1.8",
-    "@twilio/runtime-handler": "1.2.3",
```

Comment:
@GPaoloni @acedeywin This is removing `@twilio/runtime-handler` so it breaks some stuff on our serverless functions

Reply:
Looking at Twilio Serverless, this change made it downgrade to version `1.0.1` instead of `1.2.3`, and this breaks some stuff. Can you update this dependency or rollback this change?

Reply:
@murilovmachado @GPaoloni  It has been fixed. Can you check to see if anything still breaks?